lib/main_common.pm

```diff
@@ -2616,6 +2616,14 @@ sub load_publiccloud_tests {
     if (get_var('PUBLIC_CLOUD_PREPARE_TOOLS')) {
         loadtest "publiccloud/prepare_tools";
     }
+    elsif (get_var('PUBLIC_CLOUD_QAM')) {
+        my $args = OpenQA::Test::RunArgs->new();
+        loadtest "publiccloud/ssh_init", run_args => $args;
+        loadtest "publiccloud/qam_test";
```

Comment:
two lines below we have 'console/rpm' , 'console/apache' - this test names already contains some useful information which allows me to do some conclusions about what test is doing. `publiccloud/qam_test` tells me just "it is some random test for public cloud". In my understanding QAM is not about **what** to test is just about **when** . Can you come up with the name which would say **what** this test cover and not when or who is responsible for review ?

Reply:
OK, now it's `instance_overview` and there is also a better summary.